dashboard/app/reporting_email.go

```diff
@@ -339,7 +341,7 @@ func incomingMail(c context.Context, r *http.Request) error {
 		cmd.DupOf = strings.TrimSpace(strings.TrimPrefix(cmd.DupOf, replySubjectPrefix))
 		cmd.DupOf = strings.TrimSpace(strings.TrimPrefix(cmd.DupOf, emailConfig.SubjectPrefix))
 	case email.CmdUnCC:
-		cmd.CC = []string{email.CanonicalEmail(msg.From)}
+		cmd.CC = []string{email.CanonicalEmail(msg.ReplyAddress())}
```

Comment:
I think we need to use From here. Reply-To is not checked by DMARC, so it's possible to uncc other people.

Reply:
Here and below -- now it's using `From` or `X-Original-From` in case of a message coming from a mailing list.